lib/logstash/codecs/json_lines.rb

```diff
@@ -28,12 +28,21 @@ class LogStash::Codecs::JSONLines < LogStash::Codecs::Base
   # Change the delimiter that separates lines
   config :delimiter, :validate => :string, :default => "\n"
 
+  # If true, the event's metadata (the `@metadata` field) will be
+  # included when used as an output codec.
```

Comment:
Can we mark this `:deprecated` ? My hope is that we do not attempt to present metadata as a something to transmit between Logstash nodes since the intent was to keep metadata for a given event only within a single logstash instance.

Marking it deprecated lets us more easily replace this with what may be a better solution in the future.